### ISSUE TITLE
[feature] Fake advertisements on webpages in-game

### ISSUE THREAD

**Issue by CDWimmer** [HackerExperience/GameDesign#11]
Just a fun idea discussed in the development discord, perhaps they could lead to actual NPCs in the game, too. 
Terrible hand-crafted example included: 
![image](https://user-images.githubusercontent.com/31413434/29782194-89f3f5cc-8c13-11e7-988d-5316305bee12.png)

(.pdn for if someone wants to make it better: 
[downloadextraramcom.zip](https://github.com/HackerExperience/GameDesign/files/1257435/downloadextraramcom.zip)

**Comment by PedroHLC** [comment]
I do like this idea, what about you guys? @renatomassaro @kress95

**Comment by SimYouLater** [comment]
Wow, really liking this idea. Actually, the terible above example would be a great way to denote "don't click, you idiot, this is a gameplay feature that will upload an in-game virus to your in-game computer if you're stupid enough to think that the internet is safe!" to the player. Along with fake news parodies.

Better-looking ads should be safer but not always, with the risk countered by potential reward (secret in-game download of rare tool at a lower price, but the ad might also be a well-disguised link to instant malware).

Also might be possible to make player-made ad graphics that you can add to a database for them, which have to be vetted before adding but are otherwise just a matter of waiting.

Idea for an ad... "She is waiting for you, master..." (insert obligatory picture of harem girl for a facebook-parody game thing, links to either an easter egg - achievement? - or malware with a 50/50 chance )

**Comment by CDWimmer** [comment]
>terrible above example
thanks;)

I like the idea of people being able to submit adverts!
I was thinking they could lead to actual NPCs, like a clone of downloadmoreRAM.com as per the example. If those NPCs have any actual functionality above any other generic company NPC in HE that's up to devs. Perhaps dmRAM.com could actually be where you buy more server RAM? Others could just be more of the usual mission-providers, or completely useless and just there for immersion and fun, like a 'collect them all' achievement or something. In dev's spare time perhaps they could take new suggestions and add ads and more NPCs for aesthetics after the main release. Perhaps having ones for different language users would be neat. But again, things to think about after the main game is implemented.
Also remember there will be kids playing this so potentially dodgy things like "harem girls", maybe not a greeeeat idea, but funny suggestion nonetheless :P

**Comment by SimYouLater** [comment]
> Also remember there will be kids playing this so potentially dodgy things like "harem girls", maybe not a greeeeat idea, but funny suggestion nonetheless :P

Good point. Put an "answer this security question to prove you are over 18" consisting of an advanced (grade 12 level) math question. If they are able to answer it, they're either smart enough to fake any real identification well before 18 or they're old enough that it shouldn't matter. Besides, by "harem girl" I just mean the regular somewhat-skimpy silk robes outfit and no overly sexual poses, not an explicit NC-17 worthy kind of thing. Even the _actual_ Evony ads were limited in how far they could take that trick.

Toning down the language a bit might also help; "She is _waiting_, <insert player name here>..."?
